XCUITest/SearchSuggestionsTest.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  SearchSuggestionsPromptTest.swift
+//  XCUITest
+//
+//  Created by Janice Lee on 2018-10-24.
+//  Copyright © 2018 Mozilla. All rights reserved.
+//
+
+import XCTest
+
+class SearchSuggestionsPromptTest: BaseTestCase {
+    
+    override func setUp() {
+        super.setUp()
+        dismissFirstRunUI()
+    }
+    
+    override func tearDown() {
+        app.terminate()
+        super.tearDown()
+    }
+    
+    func checkToggle(isOn: Bool) {
+        let targetValue = isOn ? "1" : "0"
+        XCTAssertEqual(app.tables.switches["BlockerToggle.enableSearchSuggestions"].value as! String, targetValue)
+    }
+    
+    func checkSuggestions() {
+        // Check search cells are displayed correctly
+        let firstSuggestion = app.buttons.matching(identifier: "OverlayView.searchButton").element(boundBy: 0)
+        let secondSuggestion = app.buttons.matching(identifier: "OverlayView.searchButton").element(boundBy: 1)
+        let thirdSuggestion = app.buttons.matching(identifier: "OverlayView.searchButton").element(boundBy: 2)
+        let fourthSuggestion = app.buttons.matching(identifier: "OverlayView.searchButton").element(boundBy: 3)
+        
+        waitforExistence(element: firstSuggestion)
+        waitforExistence(element: secondSuggestion)
+        waitforExistence(element: thirdSuggestion)
+        waitforExistence(element: fourthSuggestion)
+        
+        XCTAssertEqual("g", firstSuggestion.label)
```

Comment:
If we make the change above, we would want to change all of these suggestions to make sure they just **start** with the letter g. For example:

```
        let predicate = NSPredicate(format: "label BEGINSWITH 'g'")
        let predicateQuery = app.buttons.matching(pred)
        
        // Confirm that we have at least four suggestions starting with "g"
        XCTAssert(predicateQuery.count >= 4)
```

That being said, if we think there's a reason to keep the mock suggestions that don't actually query the server, we can keep this as is.

Reply:
I do not think there would be a case where a single letter would return less than 4 suggestions, and the search suggestions must start with the letter typed (in all the engines we use), so I think that's a non-issue. 

I understand the concern of not wanting our tests to rely on a 3rd party, but as a web browser it's kind of a necessity (nearly all of our tests involve navigating to some website that could be down or change unexpectedly in the future). We'll update the tests if we ever run into an issue with that.